config/tailwind/buttons.js

```diff
@@ -21,24 +21,24 @@ module.exports = plugin(({ addComponents }) => {
 
   const buttons = {
     // Variants
-    '.btn': {
+    '#parts-kit .btn': {
       ...base,
       '@apply bg-blue-500 text-white enabled:hover:bg-blue-600 enabled:active:bg-blue-700 aria-expanded:bg-blue-700':
         {},
     },
-    '.btn-outline': {
+    '#parts-kit .btn-outline': {
       ...base,
       '@apply bg-transparent text-gray-600 border border-gray-600 enabled:hover:bg-black/5 enabled:active:bg-black/10 aria-expanded:bg-black/10':
         {},
     },
-    '.btn-subtle': {
+    '#parts-kit .btn-subtle': {
       ...base,
       '@apply bg-transparent text-gray-600 enabled:hover:bg-black/5 enabled:active:bg-black/10 aria-expanded:bg-black/10 dark:text-gray-300 dark:active:text-white dark:enabled:active:bg-black/30 dark:enabled:hover:bg-black/20 dark:aria-expanded:bg-black/30 dark:aria-expanded:text-white':
         {},
     },
 
     // Icon buttons (uses base variants for styling)
-    '.btn-icon': {
+    '#parts-kit .btn-icon': {
```

Comment:
Seems odd that the `important: '#parts-kit'` selector strategy in the config isn't working for plugins. According to their docs it should work, but maybe there's a bug in TW's plugin code.

Reply:
Yeah, I'm unsure. We were using similar naming conventions on `blueprint` for buttons and it was actually overriding the Parts Kit styles which is how I noticed the bug.

The radix issue makes sense though since that was outside of the scope of our `#parts-kit` identifier.